common/geth/handle_error.go

```diff
@@ -0,0 +1,41 @@
+package geth
+
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+// handleHttpError returns a boolean indicating if error atrributes to remote RPC
+func (f *FailoverController) handleHttpError(httpRespError rpc.HTTPError) bool {
```

Comment:
Does this assume the service provider expose gRPC interface, but using HTTP error code?
It looks quite mixed whether it's RPC or HTTP.

Reply:
fixed